coursebuilder/modules/explorer/_static/components/pages/certification.html

```diff
@@ -148,7 +148,7 @@ <h3 class="how-it-works-heading">Apply your knowledge</h3>
             <div class="item">
               <div class="row">
                 <div class="col m6 s12 how-it-works-img">
-                  <img src="/modules/explorer/_static/images/how3.png" alt="">
+                  <img src="/modules/explorer/_static/images/how3.jpg" alt="">
```

Comment:
@andela-iakande The missing images, thanks.

Reply:
@andela-angene have fixed this. https://github.com/CodeForAfricaLabs/coursebuilder-core/pull/18/commits/ac7e61efbb0c10076a5d747ed2604ef9e3563143